lib/ex_bitmex/credentials.ex

```diff
@@ -1,9 +1,43 @@
 defmodule ExBitmex.Credentials do
+  require Logger
+
   @type t :: %ExBitmex.Credentials{
           api_key: String.t(),
           api_secret: String.t()
         }
 
   @enforce_keys [:api_key, :api_secret]
   defstruct [:api_key, :api_secret]
+
+  @doc """
+  Default credentials
+
+  ## Examples
+      iex> ExBitmex.Credentials.config()
+  """
+  def config(nil) do
+    %__MODULE__{
+      api_key: System.get_env("BITMEX_API_KEY"),
```

Comment:
Please handle the case when variables are nil

Reply:
No need to handle it here. API/websocket module already handle this. It gonna handle invalid credentials there

Reply:
Great 👍